src/middleware.js

```diff
@@ -24,16 +24,20 @@ export default store => {
 
         rpc
           .call(action.name, args)
-          .then(result =>
-            store.dispatch(
-              actions.fetchSuccess({
-                name: action.name,
-                args,
-                id,
-                result,
-              }),
-            ),
-          )
+          .then(result => {
+            console.log(result)
+
+            if (result) {
```

Comment:
I'm curious, what is the response code in this case? My only question is, should this be handled in the `RPCClient` library and not here? If you think this should be handled higher up in the client library, that might be good to ship a change there first, but that's your call. If you want to bounce an idea around for that, I know Eduardo and Phil have thought a bunch about the RPC server side code and the client itself and we haven't iterated on the client library since Harrison wrote it last year :D